hooks/data.js

```diff
@@ -32,6 +32,37 @@ var Hook = base.Entity.configure({
     // next date at which this task is scheduled to run
     nextScheduledDate:  base.Entity.types.Date,
   }
+}).configure({
+  version:              2,
+  signEntities:         true,
+  properties:           {
+    hookGroupId:        base.Entity.types.String,
+    hookId:             base.Entity.types.String,
+    metadata:           base.Entity.types.JSON,
+    // task template
+    task:               base.Entity.types.JSON,
+    // pulse bindings (TODO; empty for now)
+    bindings:           base.Entity.types.JSON,
+    // timings for the task (in fromNow format, e.g., "1 day")
+    deadline:           base.Entity.types.String,
+    expires:            base.Entity.types.String,
+    // schedule for this task (see schemas/schedule.yml)
+    schedule:           base.Entity.types.JSON,
+    // access token used to trigger this task via webhook
+    triggerToken:       base.Entity.types.EncryptedText,
+    // the taskId that will be used next time this hook is scheduled;
+    // this allows scheduling to be idempotent
+    nextTaskId:         base.Entity.types.EncryptedText,
+    // next date at which this task is scheduled to run
+    nextScheduledDate:  base.Entity.types.Date,
+  },
+  migrate: function(item) {
```

Comment:
nit... just showing that you can do:
```js
migrate(item) {
  delete item.task.created;
  ...
  return item;
},
```
or you can do:
```js
migrate: (item) => {
  delete item.task.created;
  ...
  return item;
},
```

don't fix it... just saying I love es6 :)